pages/datasets/_datasetId.vue

```diff
@@ -254,6 +261,15 @@ export default {
       tabsData.push({ label: 'Images', type: 'images' })
     }
 
+    // @TODO Add logic for 3D Scaffold
```

Comment:
Is there still additional logic is missing? If so, is there a ticket or functional spec that can be referenced ?

Reply:
@clohr The purpose of this is to remove the 3D Scaffold tab until we can properly full in that data per dataset. We are waiting on the team to define how that data is stored in the dataset, and then to publish that data. Since they don't know how it will be stored in the dataest, we can't create a system that would then pull that data, which would just be `false` until those datasets were published.

This ticket is captured, and will be filled out with the correct info once they have that defined. https://app.clickup.com/t/4ybd15

Reply:
👍